runtime/bcutil/cfreader.c

```diff
@@ -1501,7 +1502,8 @@ checkPool(J9CfrClassFile* classfile, U_8* segment, U_8* poolStart, I_32 *maxBoot
 				errorCode = J9NLS_CFR_ERR_BAD_INDEX__ID;
 				goto _errorFound;
 			}
-			if (cpBase[info->slot1].tag != CFR_CONSTANT_Utf8) {
+			utf8 = &cpBase[info->slot1];
```

Comment:
same here

Reply:
I had some written some code in these 2 places that made use of the new `utf8` variable, but that code has now been removed. I guess I should remove the `utf8` variable as well and revert the code to how it was previously?

Reply:
You can just revert the code